tests/tt_eager/CMakeLists.txt

```diff
@@ -0,0 +1,87 @@
+
+set(TT_EAGER_TESTS_DTX
+    dtx/collapse_transformations
+    dtx/overlap
+    dtx/tensor
+    dtx/test_dtx_tilized_row_to_col_major
+    dtx/test_dtx
+    dtx/unit_tests
+)
+# tensor_evaluate                    # <- not called in run_tt_eager.py
+
+set(TT_EAGER_TESTS_OPS
+    # ccl/test_all_gather_utils           # <- not called in run_tt_eager.py
+    ops/test_average_pool
+    ops/test_eltwise_binary_op
+    ops/test_eltwise_unary_op
+    ops/test_softmax_op
+    ops/test_layernorm_op
+    # ops/test_moreh_adam_op                  # <- not called in run_tt_eager.py
+    # ops/test_moreh_matmul_op                # <- not called in run_tt_eager.py
+    # ops/test_moreh_layernorm_op             # <- not called in run_tt_eager.py
+    ops/test_multi_queue_api
+    ops/test_transpose_op
+    ops/test_transpose_wh_single_core
+    ops/test_transpose_wh_multi_core
+    ops/test_reduce_op
+    ops/test_bcast_op
+    ops/test_bmm_op
+    # ops/test_pad_op                         # <- not called in run_tt_eager.py
+    ops/test_tilize_op
+    ops/test_tilize_zero_padding
+    ops/test_tilize_op_channels_last
+    ops/test_tilize_zero_padding_channels_last
+    ops/test_sfpu
+    ops/test_fold_op
+)
+
+set(TT_EAGER_TESTS_TENSORS
+    tensors/test_copy_and_move
+    tensors/test_host_device_loopback
+    tensors/test_raw_host_memory_pointer
+    # tensors/test_sharded_loopback           # <- not called in run_tt_eager.py
+    tensors/test_async_tensor_apis
+)
+
+set(TT_EAGER_TESTS_INTEGRATION
+    integration_tests/test_bert
+)
+
+set(EAGER_TEST_TARGETS "")
+
+function(CREATE_EXEC TESTLIST)
```

Comment:
you can consider using .cmake files to place functions/macros there so you don't clutter CMakelists

Reply:
moved to `helper_functions.cmake` in the same directory, can also move it to a top level, but only eager tests are using functions right now

Reply:
could move this to the `cmake` folder

Reply:
sounds good to me

Reply:
yep i'm moving it to cmake folder